src/fprime_gds/executables/run_deployment.py

```diff
@@ -13,9 +13,15 @@
     GdsParser,
     ParserBase,
     StandardPipelineParser,
+    OpenMCTTelemetryPollerParser
 )
 from fprime_gds.executables.utils import AppWrapperException, run_wrapped_application
 
+import fprime_openmct 
+from fprime_openmct.config_server import register_npm_package, install_npm_package, start_npm_package
+from fprime_openmct.src.fprime_openmct.TopAppDictXMLtoOpenMCTJSON import TopologyAppDictionaryJSONifier
```

Comment:
We are going to want fprime_openmct to be a soft dependency (i.e. don't error out of `fprime-gds` if `fprime-openmct` is not installed). [Here's an example](https://github.com/fprime-community/fprime-tools/blob/77aa704873e5ad0da1d66b26f7a917cc7842e97d/src/fprime/fpp/visualize.py#L14-L17) of how we do this in fprime-tools, where fprime-visual is a soft dependency.

Reply:
fixed in the latest commit